addon/components/validation-error-field.js

```diff
@@ -6,12 +6,18 @@ var ValidationErrorField = Ember.Component.extend({
     attributeBindings: ["array", "className", "model", "field", "validation", "submitted", "delayed", "index"],
     initialize: Ember.on("init", function() {
         var field = this.get("field");
-        Ember.Binding.from("model." + field).to("fieldName").connect(this);
-        Ember.Binding.from("model." + field + "IsPrimed").to("isPrimed").connect(this);
-
         var computedKeys = ["validation", "isPrimed", "submitted", "fieldName"];
+
         if(this.get("array")) {
-            computedKeys.push("array.@each." + field);
+            var fields = field.split(",");
+            fields.forEach(function(f) {
+                computedKeys.push("array.@each." + f);
+            });
+            Ember.Binding.from("model." + fields[0] + "IsPrimed").to("isPrimed").connect(this);
+            Ember.Binding.from("model." + fields[0]).to("fieldName").connect(this);
```

Comment:
Originally I wanted to keep parity w/ the existing code but in the "if" part of the conditional it's possible we don't even need to match on this because we add the listener "model.@each.field" already so in effect we cover this situation :)

I'll see if the tests pass as expected after removing this. It would seem I should also do the same for IsPrimed then (when array is passed in)